docs/mongo_collections.md

```diff
@@ -1,21 +1,241 @@
 # Mongo Database Collections
-Mongo database collections includes a table for the following: Comments, Dockets, and Documents. Each table contains nested dictionaries. 
+Mongo database collections includes a table for the following: Comments, Dockets, and Documents. 
 
 ## Comments 
-This collections contains all the comments made on a docket. The Comments collection has: 
+This collection contains the comments made on documents and the total size is 8,198,471,680. This collection is nested with multuple dictionaries. Below is an example of a comment: 
```

Comment:
I don't think you need to include the size of each collection, since it is always growing. The examples are great though!

Reply:
I agree with @mattkosack with including the size of the collections, but other than that it looks good.